docs/reference/gpu-arch/gpu-arch-spec-overview.rst

```diff
@@ -215,27 +647,54 @@ For a more detailed explanation refer to the :ref:`specific documents and guides
 LLVM target name
   Argument to pass to clang in `--offload-arch` to compile code for the given architecture.
 VRAM
-  Amount of memory available on the GPU.
+  Amount of memory available on the GPU. Some SKUs might have a different amount of VRAM.
 Compute Units
-  Number of compute units on the GPU.
-Wavefront Size
-  Amount of work-items that execute in parallel on a single compute unit. This is equivalent to the warp size in HIP.
+  Number of compute units (CUs) on the GPU.
+Warp Size
+  Amount of work-items that execute in a lock-step fashion.
+VGPR File
+  Size of the Vector General Purpose Register (VGPR) file. Holds data used in vector instructions.
+  GPUs with matrix cores also have AccVGPRs, which are Accumulation General Purpose Vector Registers, specifically used in matrix instructions.
+SGPR File
+  Size of the Scalar General Purpose Register (SGPR) file. Holds data used in scalar instructions.
```

Comment:
> Holds data used in vector instructions.
> (...)
> Holds data used in scalar instructions

This is technically not correct. How do you define scalar instruction anyways? Usually instructions that have scalar outputs are referred to as scalar instructions, but they might still have vector *inputs*, a good example being s_readlane which reads one lane of a VGPR into an SGPR.

In the other directions "vector" instructions can also have scalar inputs, this is very common for example in comparisons where threads compare their own values to a (wavefront) uniform value.

I would focus on the fact that VGPRs are not shared between threads, while SGPRs are when describing them.